libs/features/advert-page/src/lib/features-advert-page.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { SharedAdvertDetails } from '@ppm/shared/advert-details';
+import { SharedAdvertInfo } from '@ppm/shared/advert-info';
+import { SharedLessonsAccordion } from '@ppm/shared/lessons-accordion';
+import './features-advert-page.scss';
+
+/* eslint-disable-next-line */
+export interface FeaturesAdvertPageProps {}
+
+let learnItems = ['Javascript', 'React', 'Nx', 'NestJS'];
+let preRequisites = 'Anglu ir Lietuviu kalbos';
+let title = 'React + Nest.js Project';
+let description =
```

Comment:
in feature don't use static data

Reply:
Feature is still in progress. Dynamic data is planned with another JIRA task.

Reply:
Then add TODO comment below